tests/sentry/api/endpoints/test_organization_dashboards.py

```diff
@@ -578,6 +578,43 @@ def test_add_widget_with_invalid_limit_below_minimum(self):
         assert response.status_code == 400
         assert b"Ensure this value is greater than or equal to 1" in response.content
 
+    def test_add_widget_with_fieldAliases_succeeds(self):
```

Comment:
```suggestion
    def test_add_widget_with_field_aliases_succeeds(self):
```
I know the key is actually camel-case, but we should probably use snake case for the title